matches-api/src/actions/getScheduledMatches.ts

```diff
@@ -0,0 +1,75 @@
+import { Datastore } from "@google-cloud/datastore";
+import { Match } from "../objects/Match";
+import { Request, Response } from "express";
+import axios from "axios";
+import { API_FOOTBAL_DATA_URL, API_FOOTBAL_DATA_KEY } from "../constants";
+
+export const getScheduledMatches = async (req: Request, res: Response) => {
+    const datastore = new Datastore();
+
+    const matchKind = "match";
+    const matchSource = "match-source";
+
+    try {
+
+        const matchesResponse = await axios.get(
+            `${API_FOOTBAL_DATA_URL}/matches/?status=CANCELED`,
+            { headers: { "X-Auth-Token": API_FOOTBAL_DATA_KEY } }
+        );
+
```

Comment:
I don't see yet how axios is making this easier. Otherwise the code would be:
```
        const matchesResponse = await fetch(`${API_FOOTBAL_DATA_URL}/matches/?status=CANCELED`, {
            headers: {"X-Auth-Token": API_FOOTBAL_DATA_KEY}
        });

        const matches: Array<Match> = await matchesResponse.json();

        matches.map(async (match: any) => {
```